tests/testthat/test_general.R

```diff
@@ -12,8 +12,7 @@ basque <- basque %>% mutate(trt = case_when(year < 1975 ~ 0,
                             
 test_that("SCM gives the right answer", {
 
-    syn <- augsynth(gdpcap ~ trt, regionno, year, 1975, basque, progfunc="None", scm=T)
-
+    syn <- augsynth(gdpcap ~ trt, regionno, year, basque, progfunc="None", scm=T)
```

Comment:
Can you add a test that including and not including `t_int` will still give the same thing? Would be a good way to catch any breaking changes down the line.

Reply:
Done.